types/src/account_config.rs

```diff
@@ -246,16 +234,17 @@ impl BalanceResource {
     }
 }
 
+pub const LBR_NAME: &str = "LBR";
```

Comment:
I hate to request this because I see how much refactoring pain this will cause, be: might be nicer to have an enum for this +
- a function for converting the enum to a string
- a function for converting the enum to a `TypeTag`/`StructTag`
- a `TryFrom` for `TypeTag`/`StructTag` into the enum

Reply:
I can take care of this part, as well as resolving the enum to a `Type` and piping that type to the prologue & epilogue as part of rewriting #3334.

Reply:
Just pushed an update that added a function to resolve the string to a typetag. I think this should be all that's needed?